README.md

```diff
@@ -262,17 +262,22 @@ store or to any production systems/codebases. The private key for this CA is
 intentionally made [publicly available in this
 repo](test/certs/pebble.minica.key.pem).**
 
-### CA Root Certificate
+### CA Root and Intermediate Certificates
 
-Note that the CA's root certificate is regenerated on every launch. It can be
-retrieved by a `GET` request to `https://localhost:14000/root`.
+Note that the CA's root and intermediate certificates are regenerated on every
+launch. It can be retrieved by a `GET` request to `https://localhost:14000/root`
+and `https://localhost:14000/intermediate` respectively.
 
 You might need the root certificate to verify the complete trust chain of
 generated certificates, for example in end-to-end tests.
 
+The private keys of these certificates can also be retreived by a `GET` request
+to `https://localhost:14000/root-key` and `https://localhost:14000/intermediate-key`
+respectively.
+
 **IMPORTANT: Do not add Pebble's root or intermediate certificate to a trust
 store that you use for ordinary browsing or that is used for non-testing
 purposes, since Pebble and its generated keys are not audited or held to the
-same standards as the Let's Encrypt production CA and their keys, and so are
-not safe to use for anything other than testing. Also, their private keys
-will be lost as soon as the Pebble process terminates.**
+same standards as the Let's Encrypt production CA and their keys, are exposed
+by Pebble and so are not safe to use for anything other than testing. Also,
```

Comment:
I think this somewhat breaks the whole sentence, which already was pretty long before. I think it would be better to stop the sentence after "CA and their keys" and start a new one. Or keep the old sentence and just modify the next one (or add another one).

Reply:
Indeed, it is useless to export them. And I grouped the RootCertPath with them, as it is also not an acme entrypoint.

Reply:
Alright. I rephrased this part of the README.